google/cloud/spanner/samples/samples.cc

```diff
@@ -2301,6 +2573,49 @@ void RunAll(bool emulator) {
   std::cout << "\nRunning spanner_delete_data sample\n";
   DeleteData(client);
 
+  if (run_slow_integration_tests == "yes") {
+    std::string backup_id =
+        google::cloud::spanner_testing::RandomBackupName(generator);
+
+    std::cout << "\nRunning spanner_create_backup sample\n";
+    CreateBackup(database_admin_client, project_id, instance_id, database_id,
```

Comment:
You probably want to check if these need a `!emulator` guard.

Reply:
Thanks, added too